types/parse/index.d.ts

```diff
@@ -293,10 +294,19 @@ namespace Parse {
          * @returns Promise that is resolved with base64 data
          */
         getData(): Promise<string>;
+        url(options?: { forceSecure?: boolean }): string;
+        metadata(): any;
+        tags(): any;
         name(): string;
         save(options?: SuccessFailureOptions): Promise<File>;
+        cancel(): void;
+        destroy(): Promise<File>;
         toJSON(): { __type: string, name: string, url: string };
-        url(options?: { forceSecure: boolean }): string;
+        equals(other: any): boolean;
+        setMetadata(metadata: any): void;
```

Comment:
I think that it's only valid to give an object with string keys here, right?

```suggestion
        setMetadata(metadata: Record<string, any>): void;
```